### DISCUSSION TITLE
Error getting to the builder

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3406] [Help]
Hello Everyone, 

New to budibase here and after being able to start an app I now get an error using chrome or edge that shows my connection is not secure. NET::ERR_CERT_AUTHORITY_INVALID. 

Curious if anyone else is having this issue. I have cleared my cookies and cach
![error](https://user-images.githubusercontent.com/94488623/142064761-bd3a9b8e-2865-4ee1-9de3-36ce41b58e1e.PNG)
e.

**Comment by shogunpurple** [comment]
Hi @twilson9034 are you using budibase cloud? Or is this a self hosted instance?

**Comment by twilson9034** [comment]
Budibase cloud

**Comment by twilson9034** [comment]
I can access it on my phone, just not desktop.

**Comment by shogunpurple** [comment]
Ok - well the domain is valid and it should redirect to HTTPS, so I'm not sure what is happening here. Could you tell us:
- What operating system and version you are using
- Is it the same issue in all browsers?
- does it work if you go directly to https://doallapps.budibase.app

**Reply by twilson9034** [reply]
Chrome and Edge on desktop and both are up to date.

tried opening link you sent in new window and still get a "not secure"  " your connection is not private"

I can access my account page where I can select self hosting options and delete my account.

**Reply by twilson9034** [reply]
Chrome is Version 95.0.4638.69
Edge is Version 95.0.1020.53

**Reply by twilson9034** [reply]
I was able to add the Certificate to my trusted root certificates in chrome. All certificates show as being ok and no issues, Im going to restart chrome and clear cookies and cache and see if it works.

**Comment by twilson9034** [comment]
Cleared data, restarted chrome and that fixed the original issue, now our companies firewall is blocking due to security threat. I have reported and incorrect block. so we will wait and see. agh

**Reply by twilson9034** [reply]
disconnected from corporate internet and launched corporate cell phone hot spot. Nice work around. Time to Build!

**Reply by shogunpurple** [reply]
Great to hear it! I was trying to reproduce and couldn't at all - enjoy using budibase! 

That's very useful information for future, when it comes to corporate firewalls and network rules - so thank you.

**Reply by twilson9034** [reply]
No problem. glad I could help a little.

**Reply by twilson9034** [reply]
Didnt last long, some how it caught it on my phone hotspot also.
